installers/ansible/README.md

```diff
@@ -1,7 +1,7 @@
 # Crunchy Data PostgreSQL Operator Playbook
 
 <p align="center">
-  <img src="crunchy_logo.png?raw=false" alt="Mr. Crunchy" width="150"/>
+  <img src="../../crunchy_logo.png" width="150"/>
```

Comment:
Please make the `alt="Crunchy Data"`

Reply:
I copied from the top-level readme. Would you like the same there?

Reply:
Yes.